hack/terraform-quickstart/main.tf

```diff
@@ -45,6 +67,27 @@ resource "aws_instance" "worker_node" {
     volume_type = "gp2"
     volume_size = "30"
   }
+
+  provisioner "file" {
+    source = "environment.txt"
+    destination = "/tmp/environment"
+
+    connection {
+      user = "core"
+    }
+  }
+
+  provisioner "remote-exec" {
```

Comment:
Since we expect these terraform setups to be usable to bring up a real cluster, we should probably make this conditional based on a terraform variable.

Reply:
terraform does not support conditionals per se, but I added an environment variable to select which environment to push (by default a blank file, and e2e when running in the e2e environment)

Reply:
perfect, thanks